lib.rs

```diff
@@ -481,6 +460,60 @@ struct AsyncRecord {
     kv: Box<KV + Send>,
 }
 
+impl AsyncRecord {
+    /// Serializes a `Record` and an `OwnedKVList`.
+    pub fn from(record: &Record, logger_values: &OwnedKVList) -> Self {
+        let mut ser = ToSendSerializer::new();
+        record
+            .kv()
+            .serialize(record, &mut ser)
+            .expect("`ToSendSerializer` can't fail");
+
+        AsyncRecord {
+            msg: fmt::format(*record.msg()),
+            level: record.level(),
+            location: Box::new(*record.location()),
+            tag: String::from(record.tag()),
+            logger_values: logger_values.clone(),
+            kv: ser.finish(),
+        }
+    }
+
+    /// Writes the record to a `Drain`.
+    pub fn to_drain<D: Drain>(self, drain: &D) -> Result<D::Ok, D::Err> {
```

Comment:
`to_` seems like a conversion (`to_string()`).  `log_to`?

Reply:
```suggestion
    pub fn to_log<D: Drain>(self, drain: &D) -> Result<D::Ok, D::Err> {
```

Reply:
That still looks like conversion. :)